android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
@rohanagarwal94 For fixing it you can add Marker property in MicrolocationClusterWrapper.
And set marker in onClusterItemRendered() method of ClusterRenderer.

And in map fragment use this marker to show info window. Use cancelableCallBack in animateCamera method to show info window I think you have to use handle with post Delayed method.